.gatling/example.package.conf

```diff
@@ -0,0 +1,64 @@
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
+
+gatling.enterprise.package {
+  # Consistent deployment using an existing package ID
+  # https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code/#consistent-deployment-with-id
+  # id = "00000000-0000-0000-0000-000000000000"
+  name = "My package name"
+  team = "My team name" # or ID with team = "00000000-0000-0000-0000-000000000000"
+  defaults {
+    simulation {
+      locations = [
+        {
+          name: "Europe - Paris",
+          size: 2,
+          weight: 30
+        },
+        {
+          name: "AP Pacific - Mumbai",
+          size: 2,
+          weight: 70
+        }
+      ]
+      parameters {
+        ignoreDefaults = false
+        systemProperties {
+          "com.example.prop.1" = "default value from system prop 1"
+          "com.example.prop.2" = "default value from system prop 2"
+        }
+        environmentVariables {
+          MY_SIMULATION_ENV_VAR_1 = "default value from environment 1"
+          MY_SIMULATION_ENV_VAR_2 = "default value from environment 2"
+        }
+      }
+      timeWindow {
+        rampUp = 10
+        rampDown = 10
+      }
+    }
+  }
+  simulations = [
+    {
+      # Consistent deployment using an existing simulation ID
+      # https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code/#consistent-deployment-with-id
+      # id = "00000000-0000-0000-0000-000000000001"
+      classname = "computerdatabase.ComputerDatabaseSimulation"
+    },
+    {
+      # id = "00000000-0000-0000-0000-000000000002"
+      classname = "com.example.SimulationB"
+      parameters {
+        ignoreDefaults = true
+        systemProperties {
+          "com.example.prop.1" = "override value from system prop 1"
+        }
+        environmentVariables {
+          MY_SIMULATION_ENV_VAR_1 = "override value from environment 1"
+        }
+      }
+      timeWindow {
+        rampDown = 20
+      }
+    }
```

Comment:
I would prefer a working as it is configuration, so without this, or with SimulationB (or such) existing 😅

Reply:
Actually, I would only put in the example the existing simulation in the project.